tests/common_instancetypes_test.go

```diff
@@ -0,0 +1,79 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/kustomize/api/krusty"
+
+	instancetypev1alpha2 "kubevirt.io/api/instancetype/v1alpha2"
+	sspv1beta1 "kubevirt.io/ssp-operator/api/v1beta1"
+	common_instancetypes "kubevirt.io/ssp-operator/internal/operands/common-instancetypes"
+)
+
+var _ = Describe("Common Instance Types", func() {
+	BeforeEach(func() {
+		strategy.SkipSspUpdateTestsIfNeeded()
+	})
+
+	AfterEach(func() {
+		strategy.RevertToOriginalSspCr()
+	})
+
+	FContext("operand", func() {
```

Comment:
Focused context.

Reply:
Well that's embarrassing, I'm tempted to also configure https://pre-commit.com/ to run `make lint` once https://github.com/kubevirt/ssp-operator/pull/495 lands tbh to catch silly mistakes like this.